11/umbraco-cms/reference/configuration/webroutingsettings.md

```diff
@@ -26,6 +26,33 @@ An example of a web routing config with default values, and a placeholder for ap
 }
 ```
 
+## Using the webrouting options in code
+
+You can get the config values in code like thise:
```

Comment:
A suggestion to update the wording a bit here, to make a little more clear what's going on.

```suggestion
The following is an example of you can use code to get the value for the `UmbracoApplicationUrl` configuration key:
```

Reply:
good catch @jemayn ! 😄 I've updated it now 💪